packages/core/src/autoscaling/autoscaled_pool.ts

```diff
@@ -677,9 +677,7 @@ export class AutoscaledPool {
     protected async _incrementTasksDonePerSecond(intervalCallback: () => void) {
         this._tasksPerMinute.unshift(0);
 
-        if (this._tasksPerMinute.length > 60) {
-            this._tasksPerMinute.pop();
-        }
+        this._tasksPerMinute.pop();
```

Comment:
Haha, I had a comment half written that the if was useless, but in the end I was like "it doesn't hurt anything" 😄